### PR TITLE
Update MastMissions Documentation and Tests

### PR DESCRIPTION
**Pull request by snbianco**

### CODE REVIEW

File: astroquery/mast/missions.py [hunk 1]
```diff
@@ -112,8 +112,8 @@ def query_region_async(self, coordinates, *, radius=3*u.arcmin, limit=5000, offs
 
         # basic params
         params = {'target': [f"{coordinates.ra.deg} {coordinates.dec.deg}"],
-                  'radius': radius.arcmin,
-                  'radius_units': 'arcminutes',
+                  'radius': radius.arcsec,
+                  'radius_units': 'arcseconds',
```
Comment (bsipocz) [inline astroquery/mast/missions.py:116]: wow, this is a change in server expectations, or a serious bug where in the past we queried a much smaller region than desired?

If a bug, then maybe it is worth factoring it out to a separate PR.

Comment (snbianco) [reply astroquery/mast/missions.py:116]: I think more of a change in server expectations. I realized that the API doesn't accept non-integer values for radius (another team is working on fixing this), so giving the function a radius value in arcseconds and then converting to degrees, more often than not, would error. Passing the value into the API in arcseconds just makes it more likely that the value is a whole number, but it's not a perfect fix. The change shouldn't impact the size of the region that's being queried, but it's a work around to the issue with the API until we can get it fixed.

Comment (bsipocz) [reply astroquery/mast/missions.py:116]: Oh, yes, never mind me, I didn't realize that you also send up the unit, not just the value.

**Review by bsipocz — APPROVED** [review]
Thanks!

**Pull request merged**